source-map.d.ts

```diff
@@ -78,6 +78,10 @@ export interface CodeWithSourceMap {
     map: SourceMapGenerator;
 }
 
+export interface SourceMapConsumerInitialisationOptions {
+    "lib/mappings.wasm": string;
```

Comment:
This can also be an ArrayBuffer:
https://github.com/mozilla/source-map/blob/7a263408de1b60e60b6220ca3e36d9b3e4aa047d/lib/read-wasm-browser.js#L6-L11